docs/content.md

```diff
@@ -9,6 +9,7 @@ This documentation aims to be a reference for content editors that want to make
 3. [Content Editing Tips/Info](#content-editing-tips-info)
 	* [Headings](#headings)
 	* [Frontmatter](#frontmatter)
+    * [Breadcrumbs](#breadcrumbs)
```

Comment:
could you indent this a couple of spaces to align with the other bullets? or outdent them? 💅